packages/ERTP/src/paymentLedger.js

```diff
@@ -20,14 +20,35 @@ import '@agoric/store/exported';
  * @param {Brand} brand
  * @param {AssetKind} assetKind
  * @param {DisplayInfo} displayInfo
+ * @param {ShutdownWithFailure=} optTerminateWithFailure
  * @returns {{ issuer: Issuer, mint: Mint }}
  */
 export const makePaymentLedger = (
   allegedName,
   brand,
   assetKind,
   displayInfo,
+  optTerminateWithFailure = undefined,
 ) => {
+  /** @type {ShutdownWithFailure} */
+  const destroyLedgerWithFailure = reason => {
```

Comment:
For clarity, I would prefer if we don't call this `destroyLedgerWithFailure` if it does not destroy the ledger yet. (I don't want to confuse our auditors.) Can we call this method something else and move the contents of the comment into issue 3434?